modules/installation-guide/partials/assembly_installing-che-on-openshift-4-using-cli.adoc

```diff
@@ -0,0 +1,15 @@
+:parent-context-of-installing-che-on-openshift-4-using-cli: {context}
```

Comment:
As the assembly has only one include, can we just omit it and have only the procedure?